.github/workflows/swiftlint.yml

```
@@ -10,11 +10,14 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v1
+    - name: Checkout PR branch (head)
+      uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)